src/components/Message/MessageCommerce.js

```diff
@@ -8,14 +8,14 @@ import {
   ReactionSelector as DefaultReactionSelector,
   ReactionsList as DefaultReactionsList,
 } from '../Reactions';
-import MessageRepliesCountButton from './MessageRepliesCountButton';
+import { MessageRepliesCountButton } from './MessageRepliesCountButton';
```

Comment:
can you reorder these imports by `depth` and `abc` order so it becomes more logical to search through?